src/main/resources/jnt_editUserDetails/html/editUserDetailsRows.settingsBootstrap3GoogleMaterialStyle.jspf

```diff
@@ -758,7 +758,7 @@
                     </div>
                     <div class="form-actions" style="float:right;">
                         <button type="button" class="btn btn-default" onclick="switchRow('privacy')"><fmt:message key="cancel"/></button>
-                        <button class="btn btn-primary btn-raised" type="button" onclick="updatePrivacyInformation();">
+                        <button class="btn btn-primary btn-raised" type="button" onclick="updatePrivacyInformation('${user.identifier}');">
```

Comment:
You might need to use a function named `escapeJS`

Reply:
`user.identifier` is the uuid of the user's node, it should be fine without, shouldn't it?

Reply:
I don't know, I don't like the insecure-by-default attitude of JSP. Yes an identifier should only be numbers and letters, but nothing's preventing `user` from being a malicious object? There are probably more interesting things to do when you manage to compromise the user object, but nonetheless we should promote best security practices (like we do with Spring mdr)